src/test/resources/com/puppycrawl/tools/checkstyle/filters/suppressionpatchxpathfilter/SummaryJavadocCheck/context/Test.java

```diff
@@ -0,0 +1,10 @@
+package TreeWalker.javadoc.SummaryJavadoc;  // violation without filter on line 4
+
+public class Test {
+    /**
+     *
+     */
+    public String ValidFunction(){
+        return "";
+    }
```

Comment:
`void foo(){}`

please economize space, we do not need formatting for this.

Please create one more empty javadoc method where you change name of method - there should be no violation.

Reply:
> Please create one more empty javadoc method where you change name of method - there should be no violation.

because we use `neverSuppressedChecks` in `context`, so it will be a violation.